lib/lab_manager/workers/action_worker.rb

```diff
@@ -26,8 +26,10 @@ def perform(action_id)
         when 'create_vm'
           create_vm
         when 'suspend'
-        when 'shut_down'
+        when 'shutdown'
+          shut_down
```

Comment:
please use same command name as method you are calling. There are all same expect shut_down vs shutdown